modules/account-lib/src/coin/hbar/utils.ts

```diff
@@ -112,3 +113,14 @@ export function getCurrentTime(): string {
 export function isValidTimeString(time: string) {
   return /^[0-9]+(\.[0-9]+)?$/.test(time);
 }
+
+/**
+ * Returns whether or not the string is a valid amount number
+ *
+ * @param {string} amount - the string to validate
+ * @returns {boolean} - the validation result
+ */
+export function isValidAmount(amount: string): boolean {
+  const bigNumberAmount = new BigNumber(amount);
+  return bigNumberAmount.isInteger() && bigNumberAmount.isGreaterThanOrEqualTo(0);
```

Comment:
Is there a max amount of HBARs per transaction?

Reply:
I found this on their SDK `const maxTinybar = new BigNumber(2).pow(63).minus(1);`
We could add that, the idea is to use tinyBars as input right?
(there are 100,000,000 tinyBars in one Hbar)